### ISSUE TITLE
[Alan] Scene 5 Audio - simple grammar edit

### ISSUE THREAD

**Issue by bakkenb** [w3c/wai-people-use-web-videos#208]
grammar typo "too"

Current:

> But to often the focus indicator, which highlights the currently active object on the screen, is not visible or jumps around completely out of order.

Recommended:

> But **_too_** often the focus indicator, which highlights the currently active object on the screen, is not visible or jumps around completely out of order.

**Comment by nitedog** [comment]
Addressed by #177
